src/util/normalize-manifest/fix.js

```diff
@@ -185,11 +188,17 @@ export default async function (
     if (!info.bin && binDir && typeof binDir === 'string') {
       const bin = info.bin = {};
 
-      for (const scriptName of await fs.readdir(path.join(moduleLoc, binDir))) {
-        if (scriptName[0] === '.') {
-          continue;
+      try {
```

Comment:
Could we do `await fs.exists()` here instead of the `try-catch`? Seems much nicer.